markdown/bitburner.ns.peek.md

```diff
@@ -16,7 +16,7 @@ peek(portNumber: number): PortData;
 
 |  Parameter | Type | Description |
 |  --- | --- | --- |
-|  portNumber | number | Port to peek. Must be an integer between 1 and 20. |
+|  portNumber | number | Port to peek. Must be a positive integer |
```

Comment:
`Must be a positive integer.` 

Add period  when appropriate unless it's a style necessity.

Reply:
In progress thanks for the spot